samples/Caliburn.Metro.Demo/Caliburn.Metro.Demo.csproj

```diff
@@ -45,11 +25,35 @@
     <Reference Include="Caliburn.Micro.Platform">
       <HintPath>..\..\packages\Caliburn.Micro.2.0.0\lib\net40\Caliburn.Micro.Platform.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Threading.Tasks" Condition=" '$(Configuration)' == 'Net4-Debug' Or '$(Configuration)' == 'Net4-Release' ">
```

Comment:
I guess that these conditions are hand-written. When adding additional references in future, you would always have to fix the `.csproj` files manually afterwards :unamused: